roles/update_containers/tasks/main.yml

```diff
@@ -24,6 +24,7 @@
   ansible.builtin.template:
     src: "update_containers.j2"
     dest: "{{ cifmw_update_containers_dest_path }}"
+    mode: "0755"
```

Comment:
is that one right? this looks like a file not a directory and you have used 0644 for all other files